src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -206,12 +207,14 @@ public async Task<OrchestratorResponse<ApprenticeshipDetailsViewModel>> GetAppre
 
                 apprenticeship.HashedAccountId = hashedAccountId;
 
+                var includeFrameworks = commitmentData.Commitment.ApprenticeshipEmployerTypeOnApproval != ApprenticeshipEmployerType.NonLevy;
```

Comment:
Not part of this request, but the same question as before TransferFunded cohorts should not include frameworks

Reply:
Transfers are dealt with differently anyway, so no